ledcoords/w128h32.json.example

```diff
@@ -159,8 +159,11 @@
       "name": {
         "x": 1
       },
-      "stat": {

```

Comment:
Are these changes passive to custom coords or is going to break them? I *think* it reads from the default files if it can't find a key in the custom file but I can't remember.

Reply:
This should be pretty passive for custom coords. The `stat` entry in the JSON was renamed so it doesn't exist anymore. Any custom coords that reference `stat` will just be ignored and the new `record` and `games_back` keys will be used instead. So it will require people to update their customizations, but it shouldn't crash or anything.